src/main/resources/config.sk

```diff
@@ -325,6 +331,8 @@ databases:
 		monitor changes: false
 		monitor interval: 20 seconds
 
+		backups to keep: -1
```

Comment:
You can use FlatFileStorage#load_i method to avoid requiring this on every database.